app/components/EventList.jsx

```diff
@@ -1,6 +1,16 @@
 import React from 'react';
 import EventCard from './EventCard.jsx';
 
+/**
+ * Shortens string and adds "..." at the end
+ * @param  {string} text   Original string
+ * @param  {length} length Length of string to leave
+ * @return {string}        Shortened string
+ */
+function shorten(text, length) {
+  return text.substring(0, length) + "...";
```

Comment:
@skibish there should be check if text is less then length argument.
For example text would be 160 chars, then this function will add unnecessary dots at the end.